src/Workspace/index.jsx

```diff
@@ -112,28 +113,102 @@ class Workspace extends React.Component {
     path: `${workspaceStatusUrl}`,
     method: 'GET',
   }).then(
-    ({ data }) => data.status,
+    ({ data }) => data,
   ).catch(() => 'Error');
 
-  getIcon = (notebook) => {
-    if (this.regIcon(notebook, 'R Studio')) {
+  getIcon = (workspace) => {
+    if (this.regIcon(workspace, 'R Studio')) {
       return rStudioIcon;
-    } else if (this.regIcon(notebook, 'Jupyter')) {
+    } else if (this.regIcon(workspace, 'Jupyter')) {
       return jupyterIcon;
-    } else if (this.regIcon(notebook, 'Galaxy')) {
+    } else if (this.regIcon(workspace, 'Galaxy')) {
       return galaxyIcon;
-    } else if (this.regIcon(notebook, 'DICOM')) {
+    } else if (this.regIcon(workspace, 'DICOM')) {
       return ohifIcon;
     }
     return jupyterIcon;
   }
 
+  getWorkspaceLaunchSteps = (workspaceStatusData) => {
+    if (!(workspaceStatusData.status !== 'Launching' || workspaceStatusData.status !== 'Stopped')
+    || !workspaceStatusData.conditions || workspaceStatusData.conditions.length === 0) {
+      // if status is not 'Launching', or 'Stopped',
+      // or we don't have conditions array, don't display steps bar
+      console.log(workspaceStatusData);
```

Comment:
Did you mean to keep this console log? Also -- I'm surprised the Travis is passing without an eslint rule-disable

Reply:
oops sorry I didn't push in the lates code 🤦 

I also found that eslint does not always work for some older files, probably an eslint config error